### PR TITLE
Create sample-gpt-text.json

### PR DESCRIPTION
**Pull request by mollyexten**
New version with assertion on status code and variable created from response

### CODE REVIEW

**Comment by dabelche** [comment]
Thanks Molly!

**Review by dabelche — APPROVED** [review]
Looks good to me!

File: gen-ai/sample-gpt-text.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+          ],
+          "raw": "{{api.url}}"
+        }
+      },
+      "event": [
+        {
+          "mablVariables": [
+            {
+              "description": "Response generated from the model",
+              "assertTarget": "JSONBody",
```
Comment (dabelche) [inline gen-ai/sample-gpt-text.json:73]: Weird that this is called an "assertTarget". I'm assuming that's postman stuff.

Comment (mollyexten) [reply gen-ai/sample-gpt-text.json:73]: Seems to be

**Pull request merged**